lib/classes/Utils.js

```diff
@@ -332,6 +332,7 @@ class Utils {
             isDockerContainer: isDockerContainer(),
             isCISystem: ci.isCI,
             ciSystem: ci.name,
+            isStandaloneExecutable: serverless.isStandaloneExecutable,
```

Comment:
Do we have to change the schema somehow? I don't think so. Just want to be sure that this gets recorded somehow.

Reply:
I discussed it with @alexdebrie, and we agreed that I'll just add this property, and that'll be fine

Reply:
Sounds good. Thanks! :+1: